src/node_file.cc

```diff
@@ -438,47 +438,26 @@ Local<Value> BuildStatsObject(Environment* env, const uv_stat_t* s) {
   // We need to check the return value of Number::New() and Date::New()
   // and make sure that we bail out when V8 returns an empty handle.
 
-  // Unsigned integers. It does not actually seem to be specified whether
-  // uid and gid are unsigned or not, but in practice they are unsigned,
-  // and Node’s (F)Chown functions do check their arguments for unsignedness.
+  // Numbers.
```

Comment:
This comment looks pretty out of place now. Care to just remove it?

Reply:
These comments are added on e9ce8fc82a6d12e790511b62852e820d8be03186 to accompany different `X` macro definition. I don't think it is out of place and we are good to let it be.